Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,132 @@ Meaning of columns:
+| `IfcBooleanClippingResult`            | :heavy_check_mark:   | `SolidModelConverter`      | :x:                   | -     |
+| `IfcBooleanResult`                    | :heavy_check_mark:   | `SolidModelConverter`      | :heavy_check_mark:    | -     |
+| `IfcBoundedCurve`                     | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
+| `IfcBoundingBox`                      | :heavy_check_mark:   | `SolidModelConverter`      | :x:                   | -     |
+| `IfcBoxedHalfSpace`                   | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcBSplineCurveWithKnots`            | :heavy_check_mark:   | `SplineConverter.`         | :x:                   |[#404 ](https://github.com/tumcms/Open-Infra-Platform/issues/404)|
+| `IfcBSplineSurfaceWithKnots`          | :x:                  | `SplineConverter.`         | -                     | -     |
+| `IfcCartesianPoint`                   | :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
+| `IfcCartesianPointList2D`             | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
+| `IfcCartesianPointList3D`             | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcCartesianTransformationOperator2D`| :heavy_check_mark:   | `PlacementConverter`       | :x:                   | -     |
+| `IfcCartesianTransformationOperator3D`| :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
+| `IfcCartesianTransformationOperator2DnonUniform` | :heavy_check_mark:   | `PlacementConverter`       | :x:                   | -     |
+| `IfcCircle`                           | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcCircularArcSegment2D`             | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcClosedShell`                      | :heavy_check_mark:   | `SolidModelConverter`      | :heavy_check_mark:    | -     |
+| `IfcClothoid`                         | :x:                  | `CurveConverter`           | -                     | [#304](https://github.com/tumcms/Open-Infra-Platform/issues/304) |
+| `IfcCompositeCurve`                   | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
+| `IfcCompositeCurveOnSurface`          | :x:                  | -                          | -                     | -     |
+| `IfcCompositeCurveSegment`            | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
+| `IfcConnectedFaceSet`                 | :x:                  | 'RepresentationConverter'  | -                     | -     |
+| `IfcCsgSolid`                         | :heavy_check_mark:   | `SolidModelConverter`      | :heavy_check_mark:    | [#356](https://github.com/tumcms/Open-Infra-Platform/issues/356) |
```

Comment:
This issue has nothing to do with IfcCsgSolid :)

Reply:
Resolved 3972ec7